lib/src/compile.ts

```diff
@@ -21,18 +21,21 @@ export type Syntax = 'scss' | 'indented' | 'css';
 export async function compile(options: {
   path: string;
   sourceMap?: (sourceMap: RawSourceMap) => void;
+  includePaths?: string[];
```

Comment:
Here and throughout: rename this to `loadPaths` to match the new JS API (https://github.com/sass/sass/blob/main/spec/js-api/options.d.ts#L74).